sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_options.hpp

```diff
@@ -653,27 +728,6 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     } TransferOptions;
   };
 
-  using ScheduleFileExpiryOriginType = Blobs::Models::ScheduleBlobExpiryOriginType;
-
```

Comment:
Why was this moved in the file?

Reply:
Because **CreatePathOptions** uses **ScheduleFileDeletionOptions** , so I moved  **ScheduleFileDeletionOptions** before **CreatePathOptions**.  
And both **ScheduleFileExpiryOriginType** and **ScheduleFileDeletionOptions** are parameters of ScheduleDeletion, so I think it is better to make **ScheduleFileExpiryOriginType** and **ScheduleFileDeletionOptions** define in the same place，this will be more readable.